app/helpers/statistics_helper.rb

```diff
@@ -4,13 +4,21 @@
 
 module StatisticsHelper
   def registrations_status statistics
-    if statistics.open_registrations?
+    if !!statistics.open_registrations?
```

Comment:
Or just add a `?` to https://github.com/diaspora/diaspora/blob/develop/app/presenters/statistics_presenter.rb#L34 to properly fix it.

Also add an RSpec test that triggers the bug.